ws2811.c

```diff
@@ -36,10 +37,44 @@
 #define WS2811_ZERO			(TIM_PERIOD * 0.2)
 #define WS2811_ONE			(TIM_PERIOD * 0.8)
 
+// Gamma table.
```

Comment:
Please describe in a comment the formula which generated the table.

Reply:
gamma table is (almost certainly) wrong - it is non-monotonous ... 

It may be possible to generate it using gcc inlining of floating-point functions, which will constant-fold initializer. It's not strictly standard compliant (`__extension__` can be used to silence warnings), but generated code may be easier to use/modify ... 

```
#define T(i) ((int)roundf(powf((float)i / 255.0, 1.0 / 0.45) * 255.0))
static const uint8_t gamma_table[256] = {
T(0), T(1), T(2), ... 
};
#undef T